packages/demo/usecase-demo/src/main.tsx

```diff
@@ -31,7 +31,7 @@ ReactDOM.createRoot(document.getElementById('root')!).render(
   <React.StrictMode>
     <WagmiConfig config={wagmiConfig}>
       <QueryClientProvider client={queryClient}>
-        <DevModal />
+        <DiscaimerModal />
```

Comment:
Typo to be fixed, missing one letter it seems.

```suggestion
        <DisclaimerModal />
```

Reply:
I'd put it down below, just before <Toaster />

This `<DisclaimerModal>` doesn't need `WagmiConfig` and `QueryClientProvider` so we can put it outside these providers.

Reply:
done